src/gui/DatabaseWidget.cpp

```diff
@@ -523,6 +539,15 @@ void DatabaseWidget::restoreSelectedEntries()
     }
 }
 
+void DatabaseWidget::expireEntries(QList<Entry*> selectedEntries)
+{
+    if (selectedEntries.isEmpty()) {
+        return;
+    }
+
+    GuiTools::expireEntries(this, selectedEntries);
+}
```

Comment:
This function is unnecessary

Reply:
I think I put it in there to mirror the `deleteSelectedEntries`/`deleteEntries` structure. Would you like me to look into consolidating the delete methods as well, or just expire ones?

Reply:
It just provides an indirection to calling the static Tools function. The caller would be better off just calling that function directly instead of involving the database widget in that transaction.

Reply:
I see -- I should have looked more closely at what was in there, my apologies.